src/homework13/BookManager.java

```diff
@@ -0,0 +1,86 @@
+package homework13;
+
+import java.io.*;
+import java.util.*;
+
+public class BookManager {
+
+    Map<String, Integer> words = new HashMap<>();
+    List<Words> sortedWords = new ArrayList<>();
+    final int SELECT_TOP_N = 10;
```

Comment:
if it is constant then it needs to be. -> public static final int {name of constant}